src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menArray = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    menArray = people.filter(person => {
+      const personLiveCentury = Math.ceil(person.died / 100);
+
+      return person.sex === 'm' && personLiveCentury === century;
+    });
+  }
```

Comment:
```suggestion
  const menArray = people.filter(person => century ? person.sex === 'm' : Math.ceil(person.died / 100) && person.sex === 'm' );
```
it can be shorter.
same for calculateWomenAverageAge